src/server/game/Handlers/MiscHandler.cpp

```diff
@@ -373,18 +372,18 @@ void WorldSession::HandleLogoutRequestOpcode(WorldPacket& /*recvData*/)
     else if (GetPlayer()->duel || GetPlayer()->HasAura(9454)) // is dueling or frozen by GM via freeze command
         reason = 2;                                         // FIXME - Need the correct value
```

Comment:
Maybe someone knows correct values for reasons? If so I can update this

Reply:
What about this? Anyone?

Reply:
Apparently no one knows